samples/subsys/usb/cdc_acm/sample.yaml

```diff
@@ -12,9 +13,15 @@ tests:
   sample.usb.cdc-acm.comp:
     depends_on: usb_device
     tags: usb
+    platform_exclude: native_posix native_posix_64
     extra_args: "-DOVERLAY_CONFIG=overlay-composite-cdc-msc.conf"
     harness: console
     harness_config:
       type: one_line
       regex:
         - "Wait for DTR"
+  sample.usb.cdc-acm.buildonly:
+    depends_on: usb_device
+    tags: usb
+    platform_allow: native_posix native_posix_64
```

Comment:
Do we need the platform_allow?

Reply:
I think so, for the native_posix samples should only be built and not executed in CI.

Reply:
@galak ping